### PR TITLE
feat: Scheduler BasicAbility & SDK

### PR DESCRIPTION
**Pull request by gavin1ee**

### CODE REVIEW

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Model/SchedulerJobHttpConfig.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace Masa.BuildingBlocks.BasicAbility.Scheduler.Model;
+
+public class SchedulerJobHttpConfig
+{
+    public HttpMethods HttpMethod { get; set; }
+
+    public string RequestUrl { get; set; } = string.Empty;
+
+    public List<KeyValuePair<string, string>> HttpParameters { get; set; } = new List<KeyValuePair<string, string>>();
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Model/SchedulerJobHttpConfig.cs:12]: public List<KeyValuePair<string, string>> HttpParameters { get; set; } = new ();

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Model/SchedulerJobHttpConfig.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+    public List<KeyValuePair<string, string>> HttpHeaders { get; set; } = new List<KeyValuePair<string, string>>();
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Model/SchedulerJobHttpConfig.cs:14]: public List<KeyValuePair<string, string>> HttpHeaders { get; set; } = new ();

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/_Imports.cs [hunk 3]
```diff
@@ -2,3 +2,6 @@
 // Licensed under the MIT License. See LICENSE.txt in the project root for license information.
 
 global using Masa.BuildingBlocks.BasicAbility.Scheduler.Model;
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/_Imports.cs:4]: global sort

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Enum/HttpMethods.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.BasicAbility.Scheduler.Enum;
+
+public enum HttpMethods
```
Comment (zhenlei520) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Enum/HttpMethods.cs:6]: System.Net.Http.HttpMethods?

**Review by zhenlei520 — APPROVED** [review]
@doddgu Need to pay attention to HttpMethods

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Service/ISchedulerJobService.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.BasicAbility.Scheduler.Service;
+
+public interface ISchedulerJobService
+{
+    Task<Guid> AddSchedulerJobAsync(AddSchedulerJobRequest job);
```
Comment (doddgu) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Service/ISchedulerJobService.cs:8]: Meaningless  repeat `SchedulerJob`.

File: src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Service/ISchedulerTaskService.cs [hunk 6]
```diff
@@ -0,0 +1,11 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.BasicAbility.Scheduler.Service;
+
+public interface ISchedulerTaskService
+{
+    Task<bool> StopSchedulerTaskAsync(BaseSchedulerTaskRequest request);
```
Comment (doddgu) [inline src/BasicAbility/Masa.BuildingBlocks.BasicAbility.Scheduler/Service/ISchedulerTaskService.cs:8]: 'SchedulerTask' meaningless too.

**Review by doddgu — CHANGES_REQUESTED** [review]

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**